### DISCUSSION TITLE
Failed to update prerender cache, Error: ENOSPC: no space left on device,

### DISCUSSION THREAD

**Discussion by cfftony** [vercel/next.js#57729] [Help]
### Summary

There are lots of errors :
 Failed to update prerender cache for xxxxxx [Error: ENOSPC: no space left on device, open '/data/web/.next/cache/fetch-cache/xxxxxx']

But the disk space is enough......

![image](https://github.com/vercel/next.js/assets/32441601/7a8eec9f-fed2-41a5-8d7b-3c0f84466338)

 

### Additional information

Nextjs: 13.5.6
App Router
Self hosting

**Comment by BITS-Martin** [comment]
I experienced the same problem

**Comment by BITS-Martin** [comment]
@cfftony i don't know if this is still an issue for you but we discovered this error after an upgrade from 13.5.2 to 13.5.6. Even though the disk had some free space, the `.next/cache/fetch-cache` folder had a size of 51GB and NextJS wasn't able to store more cached requests in there.
I moved and recreated the fetch-cache folder and the error disappeared.
`mv .next/cache/fetch-cache ~/backup/ && mkdir .next/cache/fetch-cache`

NextJS still had a memory leak which we solved by downgrading to 13.5.2.
